### PR TITLE
Template validations

### PR DESCRIPTION
**Pull request by phoebemirman**

### CODE REVIEW

**Review by ghost — COMMENTED** [review]
This pull request has been automatically deployed to [FeaturePeek](https://featurepeek.com). 💫

 This link will always point to this PR's latest changes: https://peek.run/r9zh3r1

File: packages/augur-ui/src/modules/common/validations.ts [hunk 1]
```diff
@@ -1,159 +1,163 @@
...
+    const errors = Array(value.length).fill('');
+    const invalid = value.findIndex(
+      outcome => outcome.toLowerCase() === INVALID_OUTCOME.toLowerCase()
+    );
+    if (invalid !== -1)
+      errors[invalid] = ['Can\'t enter "Market is Invalid" as an outcome'];
 
     let dupes = {};
-    value.forEach((outcome,index) => {
+    value.forEach((outcome, index) => {
```
Comment (bthaile) [inline packages/augur-ui/src/modules/common/validations.ts:139]: checking dup should be case insensitive.

File: packages/augur-ui/src/modules/create-market/components/form.tsx [hunk 2]
```diff
@@ -80,7 +82,10 @@ import Styles from 'modules/create-market/components/form.styles.less';
 
 import MarketView from 'modules/market/components/market-view/market-view';
 import { BulkTxLabel } from 'modules/common/labels';
-import { tellIfEditableOutcomes, createTemplateOutcomes } from '../get-template';
+import {
+  tellIfEditableOutcomes,
+  createTemplateOutcomes,
+} from '../get-template';
```
Comment (bthaile) [inline packages/augur-ui/src/modules/create-market/components/form.tsx]: need fully qualified reference to get-template

File: packages/augur-ui/src/modules/markets/reducers/new-market.ts [hunk 3]
```diff
@@ -134,6 +134,7 @@ export default function(
     }
     case RESET_STATE:
     case CLEAR_NEW_MARKET:
+      console.log(JSON.parse(JSON.stringify(EMPTY_STATE)));
```
Comment (bthaile) [inline packages/augur-ui/src/modules/markets/reducers/new-market.ts]: console log here

**Review by bthaile — APPROVED** [review]
lgtm

**Pull request merged**